src/CommunityToolkit.Maui.Core/Platform/StatusBar/StatusBar.ios.cs

```diff
@@ -137,7 +137,7 @@ static bool TryUpdateStatusBarAppearance(UIWindow? window)
 
 		if (vc is null)
 		{
-			Trace.WriteLine("Unable to update Status Bar Appearance because Current UIViewController is null");
+			Trace.TraceError("Unable to update Status Bar Appearance because Current UIViewController is null");
```

Comment:
Please do not use `TraceError` in the library. It causes crash reporting tools like sentry.io to report it as an unhandled exception. Leading to our library causing erroneous bug reports for our users.